src/js/Framework/DateTimePicker/TimePicker.js

```diff
@@ -1,10 +1,12 @@
 import flatpickr from 'flatpickr'
+import { Dutch } from 'flatpickr/dist/l10n/nl.js'
 
 export class TimePicker {
   constructor (element) {
     this.element = flatpickr(element, {
       enableTime: true,
-      noCalendar: true
+      noCalendar: true,
+      locale: Dutch
```

Comment:
Kunnen we dat niet op basis van de locale doen? As in: iets dat gaat kijken op welke taal ze zitten?

Reply:
New idea in [5d8ea78](https://github.com/sumocoders/FrameworkStylePackage/pull/110/commits/5d8ea786183aad00a9318b96df24cfafbbb46d04)

Reply:
Is deze hardcoded "Dutch" dan nog nodig?